Dependencies are not resolved

Creating a library using the template cause the nested error.

```

Installing template dependencies using npm...
npm WARN idealTree Removing dependencies.react-scripts in favor of devDependencies.react-scripts
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: testlib@0.1.0
npm ERR! Found: react@18.2.0
npm ERR! node_modules/react
npm ERR!   react@"^18.2.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
```

It persists.
Sorry, I didn't realize I'd posted an incomplete log. Here is the full one

```
/usr/local/bin/npm install
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: tslib@0.1.0
npm ERR! Found: react@18.2.0
npm ERR! node_modules/react
npm ERR!   react@"^18.2.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^16.9.0 || ^17.0.0" from @testing-library/react-hooks@8.0.1
npm ERR! node_modules/@testing-library/react-hooks
npm ERR!   dev @testing-library/react-hooks@"8.0.1" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /Users/salo/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/salo/.npm/_logs/2022-12-12T10_13_09_147Z-debug-0.log

Process finished with exit code 1
```

--- a/.github/workflows/merge-jobs.yml
+++ b/.github/workflows/merge-jobs.yml
@@ -13,22 +13,22 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - uses: actions/checkout@v2
-      - uses: c-hive/gha-yarn-cache@v1
-      - name: Cancel Previous Runs
-        uses: styfle/cancel-workflow-action@0.9.1
+      - uses: actions/checkout@v3
+      - name: Cancel Workflow Action
+        uses: styfle/cancel-workflow-action@0.10.1
         with:
           access_token: ${{ github.token }}
       - name: Read .nvmrc
         run: echo ::set-output name=NVMRC::$(cat .nvmrc)
         id: nvm
       - name: Use Node.js ${{ steps.nvm.outputs.NVMRC }}
-        uses: actions/setup-node@v2
+        uses: actions/setup-node@v3
         with:
           node-version: ${{ steps.nvm.outputs.NVMRC }}
+          cache: 'yarn'
       - name: Run yarn tasks
         run: |
-          yarn --silent
+          yarn install --frozen-lockfile
       - name: Publish NPM package
         uses: JS-DevTools/npm-publish@v1
         with:

--- a/.github/workflows/pull-request-jobs.yml
+++ b/.github/workflows/pull-request-jobs.yml
@@ -4,6 +4,8 @@
 name: Node.js CI
 
 on:
+  #  push:
+  #    branches: [ master ]
   pull_request:
     branches: [ master ]
 
@@ -13,22 +15,22 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - uses: actions/checkout@v2
-      - uses: c-hive/gha-yarn-cache@v1
-      - name: Cancel Previous Runs
-        uses: styfle/cancel-workflow-action@0.9.0
+      - uses: actions/checkout@v3
+      - name: Cancel Workflow Action
+        uses: styfle/cancel-workflow-action@0.10.1
         with:
           access_token: ${{ github.token }}
       - name: Read .nvmrc
         run: echo ::set-output name=NVMRC::$(cat .nvmrc)
         id: nvm
       - name: Use Node.js ${{ steps.nvm.outputs.NVMRC }}
-        uses: actions/setup-node@v2
+        uses: actions/setup-node@v3
         with:
           node-version: ${{ steps.nvm.outputs.NVMRC }}
+          cache: 'yarn'
       - name: Run yarn tasks
         run: |
-          yarn --silent
+          yarn install --frozen-lockfile
           yarn lint:js --quiet
           yarn lint:style --quiet
           yarn test --silent

--- a/package.json
+++ b/package.json
@@ -61,7 +61,6 @@
     "@storybook/react": "6.5.13",
     "@testing-library/jest-dom": "5.16.5",
     "@testing-library/react": "13.4.0",
-    "@testing-library/react-hooks": "8.0.1",
     "@testing-library/user-event": "14.4.3",
     "autoprefixer": "9.8.8",
     "babel-eslint": "10.1.0",

--- a/src/lib/Counter/useLogic.spec.js
+++ b/src/lib/Counter/useLogic.spec.js
@@ -1,4 +1,4 @@
-import {renderHook, act} from '@testing-library/react-hooks';
+import {renderHook, act} from '@testing-library/react';
 import useLogic from './useLogic';
 
 describe('lib > Counter > useLogic', () => {
@@ -6,7 +6,7 @@ describe('lib > Counter > useLogic', () => {
   it('renders', () => {
     /**
      * Render hook, using testing-library utility
-     * @see https://react-hooks-testing-library.com/reference/api#renderhook
+     * @see https://testing-library.com/docs/react-testing-library/api#renderhook
      */
     const {result} = renderHook(() => useLogic(initialValue));
 
@@ -15,13 +15,12 @@ describe('lib > Counter > useLogic', () => {
   it('increments value', () => {
     /**
      * Render hook, using testing-library utility
-     * @see https://react-hooks-testing-library.com/reference/api#renderhook
+     * @see https://testing-library.com/docs/react-testing-library/api#renderhook
      */
     const {result} = renderHook(() => useLogic(initialValue));
 
     /**
      * Wrap state update with act
-     * @see https://react-hooks-testing-library.com/usage/basic-hooks#updates
      * @see https://fb.me/react-wrap-tests-with-act
      */
     act(() => {

--- a/writeTemplateConfig.js
+++ b/writeTemplateConfig.js
@@ -35,7 +35,7 @@ const template = {
     style: 'lib/default.css',
     files: ['lib'],
     engines: {
-      node: '>=10',
+      node: '>=14',
     },
     private: false,
     scripts: {

